nemo_rl/distributed/worker_groups.py

```
@@ -457,6 +457,17 @@ def _create_workers_from_bundle_indices(
         # Get all placement groups
         placement_groups = self.cluster.get_placement_groups()
 
+        # Get available address and port for each worker
+        available_addresses = []
+        available_ports = []
+        for group_idx, (pg_idx, local_bundle_indices) in enumerate(bundle_indices_list):
```

Ruff (0.12.2)

Loop control variable `group_idx` not used within loop body

(B007)